module/entities/helper.js

```diff
@@ -926,106 +927,86 @@ module.exports = class EntitiesHelper {
     ) {
         return new Promise(async (resolve, reject) => {
             try {
-
-                let queryObject = {};
-
-                queryObject["$match"] = {};
-
-                if (entityIds && entityIds.length > 0) {
-                    queryObject["$match"]["_id"] = {};
-                    queryObject["$match"]["_id"]["$in"] = entityIds;
-                }
-
-                if( aclData.length > 0 ) {
-                    queryObject["$match"]["metaInformation.tags"] = 
-                    { $in : aclData };
+                let entityDocuments = [];
+                let bodyData={
+                    "type" : entityType
+                };
+                
+                let entitiesData = await sunbirdService.learnerLocationSearch( bodyData );
+                
+                if( !entitiesData.data.response.length > 0 ) {
```

Comment:
@VISHNUDAS-tunerlabs check with success value and check "entitiesData.data" exist